src/fides/api/alembic/migrations/versions/75bb9ee843f5_add_soft_delete_for_privacy_request.py

```diff
@@ -0,0 +1,44 @@
+"""Adds deleted_at and deleted_by to PrivacyRequest
+
+Revision ID: 75bb9ee843f5
+Revises: 9de4bb76307a
+Create Date: 2024-09-19 14:35:30.510909
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "75bb9ee843f5"
+down_revision = "9de4bb76307a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "privacyrequest",
+        sa.Column("deleted_at", sa.DateTime(timezone=True), nullable=True),
+    )
+    op.add_column("privacyrequest", sa.Column("deleted_by", sa.String(), nullable=True))
+    op.create_foreign_key(
+        "privacyrequest_deleted_at_key",
+        "privacyrequest",
+        "fidesuser",
+        ["deleted_by"],
+        ["id"],
+        ondelete="SET NULL",
+    )
```

Comment:
ah that's a good idea I think, it may be overkill to have the FK constraint anyways -- I was following what was done for the approved_by column (I think), but it could be argued that this is a different use case anyways